### PR TITLE
Update the client to newest schema version

### PR DESCRIPTION
**Pull request by jerzy-dudek-form3**
Updating to the latest changes from Swagger schema.

Not sure if these are breaking changes, each team would have to confirm this. In case of any breaking change, we need to bump version to v6.

Co-authored-by: @georgian-micsa-form3

### CODE REVIEW

**Review by sergio-goncalves-form3 — DISMISSED** [review]

File: pkg/generated/models/payment.go [hunk 1]
```diff
@@ -1360,6 +1360,9 @@ type PaymentAttributesBeneficiaryParty struct {
 	// account with
 	AccountWith *BeneficiaryDebtorAccountHoldingEntity `json:"account_with,omitempty"`
 
+	// Additional address line of the beneficiary address
```
Comment (miketonks-form3) [inline pkg/generated/models/payment.go:1363]: Changes in this file look good for TCH-RTP

**Comment by miketonks-form3** [comment]
From the perspective of TCH-RTP Project, this looks good.

Updated payment model includes the new fields withing `BeneficiaryParty` and DebtorParty` that we added some time ago, and have been waiting for these to update to the OSS Client.

**Review by miketonks-form3 — APPROVED** [review]

**Comment by miketonks-form3** [comment]
Can this be merged as we're waiting for the updated client library in TCH-RTP project?

**Comment by jerzy-dudek-form3** [comment]
> Can this be merged as we're waiting for the updated client library in TCH-RTP project?

* we're missing approval at this point
* other teams have not really looked at it, it seems ...

**Comment by ryangough-f3** [comment]
@jerzy-dudek-form3 I think this is definitely a breaking change due to the renaming of Balances -> LHV Gateway and I think even just the addition of swagger error responses which is effectively changing the type of the response in some runtime situations.

**Comment by jerzy-dudek-form3** [comment]
Closing the PR, since we need to have a larger in-company discussion how to deal with these kind of changes.

CC @george-koulouris-form3

**Comment by jerzy-dudek-form3** [comment]
Reopening as we now have more people to collaborate on this

**Comment by ryangough-f3** [comment]
Created a new PR as it seemed easier to manage that way, and to update the source of the swagger file based on the new docs: https://github.com/form3tech-oss/go-form3/pull/70/files

**Pull request closed without merging**